src/main/java/frc/robot/subsystems/shooter/RobotMap.java

```diff
@@ -4,17 +4,19 @@
 
     public interface CAN {
 
+        int SHOOTER_TALON_ONE = 1;
+        int SHOOTER_TALON_TWO = 2;
     }
-    
+
```

Comment:
I have no idea why lines 10,14,18 and 20 are marked as changes, but un-change them

Reply:
Accidentally updated inside the branch, that's the way it's supposed to be